frontend/src/locales/en.json

```
@@ -224,6 +230,7 @@
   "navigation": {
     "admin_area": "Admin Area",
     "community": "Community",
+    "info": "Info",
```

```suggestion
    "info": "Information",
```

Is much better in my eyes!